samples/bigQueryTutorial.py

```diff
@@ -0,0 +1,13 @@
+from local_data_platform.store.source.gcp.bigquery import BigQuery  
+from local_data_platform.real_world_use_cases.near_data_lake.config.sample_queries.near_transactions import NEAR_TRANSACTION_QUERY
+from pathlib import Path
+import os
+
+"""
+creadentials.json is the path to the credentials file that you downloaded from GCP.
+"""  
+local_gcp_credential_path =   Path(os.getcwd()+'/local-data-platform/samples/credentials.json')
+store = BigQuery(credential_path=local_gcp_credential_path)
+
+df = store.query(NEAR_TRANSACTION_QUERY.query)
```

Comment:
Hi @mrutunjay-kinagi please let me know how you are testing the code. Are you using below script?
https://github.com/tusharchou/local-data-platform/blob/main/local-data-platform/tests/test_gcp_connection.py

Hope this is also updated as I am seeing another function in class BiqQuery doing the same.

Reply:
I've made changes to the the big query integration in this PR , I'll update the test in upcoming commits.